src/atom/core/config.rs

```diff
@@ -106,13 +101,6 @@ impl Dns {
     pub fn update_ip(&self) {
         // todo!()
     }
-
-    /// Create a default [`Dns`] for testing purposes.
-    fn test_value() -> Dns {
-        let code = "eWBRKL3JuN4W6KDj2vLkt94o".to_string();
```

Comment:
shit i just realized i need to invalidate that code now because it's in the git history lmao